library/src/test/java/com/datatorrent/lib/io/fs/FileSplitterBaseTest.java

```diff
@@ -97,7 +97,13 @@ protected void starting(org.junit.runner.Description description)
     @Override
     protected void finished(Description description)
     {
-      TestUtils.deleteTargetTestClassFolder(description);
+      try {
+        //got RuntimeException: java.io.IOException: Unable to delete directory target/com.datatorrent.lib.io.fs.FileSplitterBaseTest/testSplitterInApp
+        //in build server and sometime in local machine
+        TestUtils.deleteTargetTestClassFolder(description);
```

Comment:
This is in fact is not related to this task.
Maybe it related to "APEXMALHAR-1991"
But I got the exception at least twice in build server, and got once in my local machine. 
It's better to separate this issue. but this task is out of time.
May I can create a bug and merge this task even if this are problem in build server